docs/arch/lexicon.md

```diff
@@ -7,4 +6,3 @@
 - **Agent pool**: the set of all registered agents. It is a scheduler's entire knowledge about available computing resources.
 - **Pipeline**: a set of actions to be executed. It is declared as a YAML file.
 - **Scheduling**: selection of an agent on which to execute an action.
-
```

Comment:
Please re-add this removed empty line at the end of this .md file

No blank lines at the end of a file is not up to markdown standards,

And generally, it is best to leave one blank line at the end of any file (source code included), as many older tools won't work properly if a line feed isn't present. Also it is a POSIX standard, as it is the definition of a line: https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206